yb-voyager/src/queryparser/helpers.go

```diff
@@ -4,6 +4,14 @@ import (
 	"google.golang.org/protobuf/reflect/protoreflect"
 )
 
+const (
+	DOCS_LINK_PREFIX        = "https://docs.yugabyte.com/preview/yugabyte-voyager/known-issues/"
+	POSTGRESQL_PREFIX       = "postgresql/"
+	ADVISORY_LOCKS_DOC_LINK = DOCS_LINK_PREFIX + POSTGRESQL_PREFIX + "#advisory-locks"
```

Comment:
You can use the same constants from `cmd` package to keep base links at a place.

Reply:
that will probably result in cycle dependency, otherwise would have used the same place.

Reply:
okay, got it.